### PR TITLE
Explosive damage changes

### PR DESCRIPTION
**Pull request by NotDeirun**
This makes explosive damage much more deadly. I also stopped frag grenades from exploding, stopped HE grenades from spawning invisible fragments and made them a child of /grenade themselves.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -139,29 +139,13 @@
...
-	if(update)
-		UpdateDamageIcon()
-
+		
+	var/organ_hit = BP_CHEST //Chest is hit first
+	var/exp_damage
+	while (b_loss > 0)
+		b_loss -= exp_damage = rand(0, b_loss)
+		src.apply_damage(exp_damage, BRUTE, organ_hit)
+		organ_hit = pickweight(list(BP_HEAD = 0.2, BPBP_GROIN = 0.2, BP_R_ARM = 0.1, BP_L_ARM = 0.1, BP_R_LEG=0.1, BP_L_LEG=0.1))  //We determine some other body parts that should be hit
```
Comment (TorinoFermic) [inline code/modules/mob/living/carbon/human/human.dm]: Both lines are in wrong order.

Comment (TorinoFermic) [inline code/modules/mob/living/carbon/human/human.dm]: At bottom of code, it's missing UpdateDamageIcon()

Comment (NotDeirun) [reply code/modules/mob/living/carbon/human/human.dm]: From what I can tell apply_damage() already does the UpdateDamageIcon() part.
In the case that I am wrong: Should I put UpdateDamageIcon() inside or outside the while() loop?

Comment (TorinoFermic) [reply code/modules/mob/living/carbon/human/human.dm]: I am seeing these lines inside apply_damage, that's good.

**Comment by NotDeirun** [comment]
Luc𝒂rio told me to me to increase if (prob(x)) inside proc/fragment_explosion from x = 20 to x=95. I did just that.

**Pull request merged**